maro/utils/exception/cli_exception.py

```diff
@@ -121,3 +121,10 @@ def __init__(self, cli_command: str, message: str = None):
 
     def __str__(self):
         return f"command: {self.cli_command}\n {self.strerror}"
+
+
+class ProcessInternalError(CliError):
+    """ Errors in MARO CLI process mode. """
+
+    def __init__(self, message: str = None):
+        super().__init__(error_code=3999, message=message)
```

Comment:
I designed a 3-layers exception structure for CLI, and the first(CliError) and the second layer(UserFault, ClientError, ServiceError) should not be used directly.
I think ProcessInternalError is a third layer exception and should be under ClientError / SerivceError, with the specific error codes (31XX / 32XX).

Correct me if I misunderstood or comment it below if you have any concern.

FYI. CommandExecutionError is an obsoleted implementation and will be deleted in the future.